README.md

```diff
@@ -0,0 +1,33 @@
+# Hello Microverse
+
+> This is a simple Hello Miroverse Project.
+
+
+Additional description about the project and its features.
+
+## Built With
+
+- HTML- Hypertext Markup Language
+- CSS - Cascading Style Sheets
+
```

Comment:
- [ ] It's so much better and professional if you provide steps about how to clone and start your project. Please provide an explanation about how to do this.
Remember that not only developers are going to see your repo so it so much better if you explain to non developer people how to run your project. 
 👍 🥇